src/main/java/com/itschool/eventmanagment/services/EventServiceImp.java

```diff
@@ -24,4 +35,30 @@ public List<EventDTO> updateEvent() {
     public List<EventDTO> getEvents() {
         return null;
     }
-}
+
+    @Override
+    public List<DetailsParticipantDTO> getRegisteredParticipants(Long eventId) {
+        Optional<Event> eventOptional = eventRepository.findById(eventId);
```

Comment:
If our event doesn't exist, we should return a custom exception with a proper message to inform the caller. We can have something like:

```suggestion
         Event event = eventRepository.findById(eventId)
        .orElseThrow(() -> new EventNotFoundException("Event not found with id: " + eventId));
```
**HTTP Status Code** for the above example could be 404.

**Custom exception example:**

```
public class EventNotFoundException extends RuntimeException {
    public EventNotFoundException(String message) {
        super(message);
    }
}
```

I advise you (you as a team) to consider exception handling for the project; in the future, this project will need a global exception handler and custom exceptions, like the ones created in the course. There will be plenty of times when we'll want to throw custom exceptions for particular cases (like the one from above), and it will be handy to have them around.